### ISSUE TITLE
About training the entity pair encoder

### ISSUE THREAD

**Issue by Frankie123421** [Ac-Zyx/RoCORE#6]
Hello, I notice that in your code you use bert as the entity pair encoder and only set "require_grad" of the parameters of Layer 7 (default) as True. So I wonder why you don't train the whole bert, does it hurt the performance? Thank you.

**Comment by Frankie123421** [comment]
And it seems that the algorithm provided in the paper is inconsistent with the code. Firstly, in the pre-training stage, the loss in your code is L_c instead of l, and both the entity pair encoder  /phi and relation-oriented clustering module /theta will be updated.  Then, in the iteratively training stage, the same problem appears in the "optimize clustering" for both /phi and /theta will be updated, the loss is correct here. I don't know if I miss something in the code/paper, hope you can clarify it, thank you.

**Comment by weichaen** [comment]
我也发现同样的问题
